src/components/CameraPermissions/withPermissionsFlow.tsx

```diff
@@ -92,7 +93,10 @@ export default <P extends Props>(
       return (
         <PermissionsPrimer
           {...{ trackScreen, audio }}
-          onNext={this.setPermissionsPrimerSeen}
+          onNext={trackClick(
```

Comment:
This will go for all the screens that use it, not only face selfie. Can you replace it with a generic name?